update setup.py

nickmoreton opened the pull request:
# Create package file 

This PR only changes the version reference to 0.1.3 and updates the documentation for the install process.
It's now been rebased on the current main branch because it was out of date.

The package has been uploaded to PyPi: https://pypi.org/project/wagtail-wordpress-import/

Ticket URL: https://projects.torchbox.com/projects/wordpress-to-wagtail-importer-package/tickets/61

---

- Testing
    - [x] CI passes
    - [x] If necessary, tests are added for new or fixed behaviour
    - [x] These changes do not reduce test coverage
- Documentation.
    - [x] This PR adds or updates documentation
- Import.
    - [x] Tested against: https://raw.githubusercontent.com/WPTT/theme-unit-test/master/themeunittestdata.wordpress.xml with one caveat: In the xml is one page item for url https://wpthemetestdata.wordpress.com/2009/09/05/edge-case-no-title/ which has no title. Removing this fixture is required. We always expect a page title.

nickmoreton committed:
update setup.py

I was having difficulty uploading to pypi. Having checked other Wagtail packages I can see that the classifiers needed updating to use only Wagtail v2

nickmoreton committed:
update version

I could not overwrite the previous version so adding a new one v0.1.1

nickmoreton committed:
move default_prefilters() into wordpress.py

nimasmi commented on wagtail_wordpress_import/importers/wordpress.py:
```diff
@@ -304,6 +303,23 @@ def connect_page_categories(self, page, category_model, item):
             page.categories = page_categories
 
 
+def default_prefilters():
```
1. **Comment**: As discussed elsewhere, I agree this should change from a function to a constant.
2. **Question**: I also commented about functions vs constants at https://github.com/torchbox/wagtail-wordpress-import/pull/106. I'm confused about which conversation was which. Was I wrong there?

nimasmi reviewed (changes requested):
There is one change requested, as discussed elsewhere.

nickmoreton committed:
change imports to relative locations

I noticed that when the package is installed from pypi the absolute imports cannot be found

nickmoreton committed:
Revert "change imports to relative locations"

This reverts commit 95f31b61f3e5bc198a832f4ae138d6b62cf0e3d5.

nickmoreton committed:
change version

nickmoreton committed:
add pip install from pypi
